bson/bsoncodec/struct_codec.go

```diff
@@ -440,7 +491,12 @@ func (bi byIndex) Less(i, j int) bool {
 	return len(bi[i].inline) < len(bi[j].inline)
 }
 
-func (sc *StructCodec) describeStruct(r *Registry, t reflect.Type) (*structDescription, error) {
+func (sc *StructCodec) describeStruct(
+	r *Registry,
+	t reflect.Type,
+	useJSONStructTags bool,
+	errorOnDuplicates bool,
+) (*structDescription, error) {
```

Comment:
Either is valid. I find wrapped function signatures difficult to read because the wrapped function signature lines are at the same indentation level as the function body, so I prefer stacked.

Do you feel strongly one way or another?

Reply:
No I don't feel strongly either way, I just mentioned it because it's what we typically do in the driver. They both look fine to me.